app-admin/genromfs/genromfs-0.5.7.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Please 8 ;-)

Reply:
ohh, thx for pointing that out, missed that :)